docs/docs/configuration/auth.md

```diff
@@ -87,6 +88,21 @@ Note: The user is checked against the group members list on initial authenticati
 
 Note: When using the Azure Auth provider with nginx and the cookie session store you may find the cookie is too large and doesn't get passed through correctly. Increasing the proxy_buffer_size in nginx or implementing the [redis session storage](sessions.md#redis-storage) should resolve this.
 
+### Azure Auth Provider
+
+1. Open the AD FS administration console on your Windows Server and add a new Application Group
+2. Provide a name for the integration, select Server Application from the Standalone applications section and click Next
+3. Follow the wizard to get the client-id, client-secret and configure the application credentials
+4. Configure the proxy with
+
+```
+   --provider=adfs
+   --client-id=<application ID from step 3>
+   --client-secret=<value from step 3>
+```
+
+Note: When using the Adfs Auth provider with nginx and the cookie session store you may find the cookie is too large and doesn't get passed through correctly. Increasing the proxy_buffer_size in nginx or implementing the [redis session storage](sessions.md#redis-storage) should resolve this.
```

Comment:
Is this still any issue for Azure/ADFS? We moved to compressing cookies & refactored the encoding scheme a few version ago to cut their size in half, I don't think we've been hitting the 8K header limit for Azure users anymore.

Reply:
When not using redis, the cookie is divided in 2. If you add some custom header as the access_token or id_token, you can reach the 8K limit. At least in my case this note in the docs helped me to diagnose the issue since the errors thrown by the reverse proxy were not meaningful at all.